src/wpt/harness/harness.ts

```diff
@@ -628,6 +653,15 @@ globalThis.subsetTestByKey = (
   return testType(testCallback, testMessage);
 };
 
+globalThis.subsetTest = (testType, testCallback, testMessage): void => {
```

Comment:
Shouldn't this be named testSubset?

```suggestion
globalThis.testSubset = (testType, testCallback, testMessage): void => {
```

Reply:
Nope, I have to match the function name WPT is using which is `subsetTest`.